puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/release/source/foreman-installer-source-release.yaml

```diff
@@ -12,6 +12,6 @@
       - github
     dsl:
       !include-raw:
-        - pipelines/test/foreman-installer-develop-test.groovy
+        - pipelines/release/source/foreman-installer.groovy
```

Comment:
This needs to include the groovy file which defines `generate_sourcefiles` as well. Applies to `*-source-release.yaml`

Reply:
Good catch. Updated.